pkg/appStore/installedApp/service/EAMode/InstalledAppDBService.go

```diff
@@ -533,3 +535,12 @@ func (impl *InstalledAppDBServiceImpl) MarkInstalledAppVersionModelInActive(inst
 	}
 	return nil
 }
+
+func (impl *InstalledAppDBServiceImpl) IsInstalledAppManagedByArgoCd(appId int) (bool, error) {
+	installedApp, err := impl.InstalledAppRepository.GetInstalledAppsMinByAppId(appId)
```

Comment:
deployment app type is deprecated, use deploymentConfigService instead

Reply:
done